content/applications/hr/employees/new_employee.rst

```diff
@@ -0,0 +1,269 @@
+==================
+Add a new employee
+==================
+
+When a new employee is hired, the first step is to create a new employee form. Starting in the
+:menuselection:`Employees` app default view, click the :guilabel:`Create` button to access a new
+employee form. Fill out the required information (underlined in bold) and any additional details,
+then click :guilabel:`Save`.
+
+.. image:: new_employee/employee-new.png
+   :align: center
+   :alt: Create a new employee card.
+
+.. note::
+   The current company phone number and name is populated in the :guilabel:`Work Phone` and
+   :guilabel:`Company` fields.
+
+General information
+===================
+
+Required fields
+---------------
+
+- :guilabel:`Name`: Enter the employee's name.
+- :guilabel:`Company`: Select the company from the drop-down menu that the new employee is hired by,
+  or create a new company by typing the name in the field.
+- :guilabel:`Working Hours`: In the :guilabel:`Work Information` tab, select the desired
+  :guilabel:`Working Hours` from the drop-down menu.
+
+.. image:: new_employee/working-hours.png
+   :align: center
+   :alt: Working Hours are located in the Work Information tab.
+
+.. note::
+   :guilabel:`Working Hours` are related to a company's working times, and an employee cannot have
+   working hours that are outside of a company's working times.
+
+   Each individual working time is company-specific, so for multi-company databases, each company
+   needs to have its own working hours set.
+
+   If an employee's working hours are not configured as a working time for the company, new working
+   times can be added, or existing working times can be modified. To add or modify a working time,
+   go to the :menuselection:`Payroll app --> Configuration --> Working Times`, and add a new working
+   time or edit an existing one.
+
+   After the new working time is created, set the working hours for the employee.
+
+Optional fields
+---------------
+
+- :guilabel:`Photo`: In the top right image box of the employee card, click on the :guilabel:`✏️
+  (pencil)` edit icon to select a photo to upload.
+- :guilabel:`Job Position`: Enter the employee's job position title.
+- :guilabel:`Tags`: Click on a tag in the drop-down menu to add any tags applicable to the employee.
+  Any tag can be created in this field by typing it in. Once created, the new tag is available for
+  all employee cards. There is no limit to the amount of tags that can be added.
+- Work Contact Information: Enter the employees :guilabel:`Work Mobile`, :guilabel:`Work Phone`,
```

Comment:
checking— does this get a gui?

Reply:
For consistency I would like it to, because you do click on the tag, but since "tag" is generic- I removed t. The argument does stand- the word "tag" is not clickable, the actual tag itself is (the individually created ones).